tests/framework/installer/ceph_manifests.go

```diff
@@ -26,7 +26,8 @@ import (
 
 type CephManifests interface {
 	GetRookCRDs() string
-	GetRookOperator(namespace string) string
+	GetRookOperator(namespace, enableCSI string) string
```

Comment:
If CSI is always enabled, I think these can be reduced to `GetRookOperator` again, removing the need for `GetCephCSITemplate`.